Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        static string name = GetName();
+        static int age = GetAge();
+        static float weight = GetWeight();
+        static float height = GetHeight();
+        static float bmi = GetBMI(weight, height);
+
+        public static void Demo()
+        {
+            Console.WriteLine(name + " is " + age + " years old, their weight is " + weight + " kg, and their height is " + height + " cm.");
```

Comment:
Getting input and calculating it should be done in demo function and not the class level. Especially since you also need to do the same for person2. Right now, you would loose bmi of person1.

Reply:
That's a very good point, I hadn't even considered that. Alright, duly noted and code fixed.